### ISSUE TITLE
secret should be set

### ISSUE THREAD

**Issue by ituageorge** [ituageorge/exam-timed-backend-app#1]
Error: secret should be set
    at module.exports (C:\Users\GEORGE\Desktop\exam-timed-backend-app\node_modules\express-jwt\lib\index.js:20:42)
    at jwt (C:\Users\GEORGE\Desktop\exam-timed-backend-app\_helpers\jwt.js:13:12)
    at Object.<anonymous> (C:\Users\GEORGE\Desktop\exam-timed-backend-app\server.js:15:9)
    at Module._compile (internal/modules/cjs/loader.js:955:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
    at Module.load (internal/modules/cjs/loader.js:811:32)
    at Function.Module._load (internal/modules/cjs/loader.js:723:14)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1043:10)
    at internal/main/run_main_module.js:17:11
[nodemon] app crashed - waiting for file changes before starting...

**Comment by Adebolavictor** [comment]
I cant. see the `config.json` file

**Comment by Adebolavictor** [comment]
what value type is expected for secret in `jwt (C:\Users\GEORGE\Desktop\exam-timed-backend-app_helpers\jwt.js:13:12)`

**Comment by Adebolavictor** [comment]
are you there?

**Comment by Adebolavictor** [comment]
String or JSON?

**Comment by ituageorge** [comment]
string

**Comment by Adebolavictor** [comment]
put the value in the config file directly into the `secret` param of the function and see if it works

**Comment by ituageorge** [comment]
goodafternoon
The secret is working now

**Comment by Adebolavictor** [comment]
How did you resolve it?

**Comment by ituageorge** [comment]
I omitted to put require('rootpath')() at the beginning of the file to
require or import the file name directly without having to trace the path
Or simply require the file by using the path eg const jwt = require(
'../_helpers/jwt');

**Comment by ituageorge** [comment]
This is showing at the console:
Server listening on port 80
(node:11132) UnhandledPromiseRejectionWarning:
MongooseServerSelectionError: Could not connect to any servers in your
MongoDB Atlas cluster. One common reason is that you're trying to access
the database from an IP that isn't whitelisted. Make sure your current IP
address is on your Atlas cluster's
IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/
    at NativeConnection.Connection.openUri
(C:\Users\GEORGE\Desktop\exam-timed-backend-app\node_modules\mongoose\lib\connection.js:832:32)
    at
C:\Users\GEORGE\Desktop\exam-timed-backend-app\node_modules\mongoose\lib\index.js:345:10
    at
C:\Users\GEORGE\Desktop\exam-timed-backend-app\node_modules\mongoose\lib\helpers\promiseOrCallback.js:31:5
    at new Promise (<anonymous>)
    at promiseOrCallback
(C:\Users\GEORGE\Desktop\exam-timed-backend-app\node_modules\mongoose\lib\helpers\promiseOrCallback.js:30:10)
    at Mongoose._promiseOrCallback
(C:\Users\GEORGE\Desktop\exam-timed-backend-app\node_modules\mongoose\lib\index.js:1135:10)
    at Mongoose.connect
(C:\Users\GEORGE\Desktop\exam-timed-backend-app\node_modules\mongoose\lib\index.js:344:20)
    at Object.<anonymous>
(C:\Users\GEORGE\Desktop\exam-timed-backend-app\_helpers\db.js:6:10)
    at Module._compile (internal/modules/cjs/loader.js:955:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
    at Module.load (internal/modules/cjs/loader.js:811:32)
    at Function.Module._load (internal/modules/cjs/loader.js:723:14)
    at Module.require (internal/modules/cjs/loader.js:848:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous>
(C:\Users\GEORGE\Desktop\exam-timed-backend-app\users\userService.js:5:12)
    at Module._compile (internal/modules/cjs/loader.js:955:30)
(node:11132) UnhandledPromiseRejectionWarning: Unhandled promise rejection.
This error originated either by throwing inside of an async function
without a catch block, or by rejecting a promise which was not handled with
.catch(). (rejection id: 1)
(node:11132) [DEP0018] DeprecationWarning: Unhandled promise rejections are
deprecated. In the future, promise rejections that are not handled will
terminate the Node.js process with a non-zero exit code.

**Comment by Adebolavictor** [comment]
Open a new issue ticket for it

**Comment by ituageorge** [comment]
Ok. This particular issue has been resolved

**Comment by Adebolavictor** [comment]
It's important to share how it was resolved for reference purpose and to help others when they come across the issue
